templates/documents/general_upload.php

```diff
@@ -1,5 +1,5 @@
 <form method=post enctype="multipart/form-data" action="<?php echo $this->form_action;?>" onsubmit="return top.restoreSession()">
-<input type="hidden" name="MAX_FILE_SIZE" value="64000000" />
```

Comment:
This feature is not only used for little patient photos, it is also used for uploading things like echocardiogram study videos.  Assuming the 6400 is 6.4K, that will not fly.